src/test/ui/lint/not_found.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2014–2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#[allow(foo_bar)]
```

Comment:
I would expect to see this be `#[allow(FOO_BAR)]`, to show that the suggestion does not fire for nonsense lints...is there a reason you chose `foo_bar`?

Reply:
This is testing the default behavior as a baseline.